app/layouts/default.tsx

```diff
@@ -1,24 +1,27 @@
 import { T, useT } from "@transifex/react";
 import { Star } from "lucide-react";
 import { NavLink, Outlet } from "react-router";
+import FooterLink from "~/components/FooterLink";
 
 export default function Layout() {
   const t = useT();
 
   return (
     <div className="min-h-screen grid grid-cols-1 grid-rows-[auto_1fr_auto]">
       <header className="border-b border-gray-200 px-4 md:px-10 flex items-center h-[80px]">
-        <h1 className="text-xl font-bold flex gap-4 items-center relative">
-          <Star />
-          <T _str="a11y-Score" />
-          <span className="bg-gray-200 text-gray-900 leading-[10px] text-[10px] font-bold uppercase tracking-widest pl-1 pr-0.5 pt-0.5 pb-0.25 rounded absolute top-full right-0">
-            <T _str="beta" />
-          </span>
-        </h1>
+        <NavLink to="/" className="hover:text-primary">
+          <h1 className="text-xl font-bold flex gap-4 items-center relative">
+            <Star />
+            <T _str="a11y-Score" />
+            <span className="bg-gray-200 text-gray-900 leading-[10px] text-[10px] font-bold uppercase tracking-widest pl-1 pr-0.5 pt-0.5 pb-0.25 rounded absolute top-full right-0">
```

Comment:
Suggestion (non-blocking): Is there a good IDE/tooling/autocompletion for these classnames that you can recommend in the docs? If it's not much effort (<30m), could we enforce a canonical class order here to reduce merge conflicts?